clang/include/clang/Basic/arm_sve_sme_incl.td

```diff
@@ -227,6 +227,7 @@ def IsPreservesZA             : FlagType<0x10000000000>;
 def IsReadZA                  : FlagType<0x20000000000>;
 def IsWriteZA                 : FlagType<0x40000000000>;
 def IsReductionQV             : FlagType<0x80000000000>;
+def IsStreamingOrSVE2p1       : FlagType<0x80000000000>; // Use for intrinsics that are common between sme/sme2 and sve2p1.
```

Comment:
Hey Sam, 
Just in case, this is just a nit.
But maybe  we should have IsStreamingOrSVE instead of IsStreamingOrSVE2p1.
Because we can use the same flag if we have more instructions under a different version of  sve.